platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/HttpSolrClientFactory.java

```diff
@@ -183,7 +178,7 @@ public static String getDefaultHttpsAddress() {
     }
   }
 
-  private static SSLContext getSslContext() {
+  static SSLContext getSslContext() {
```

Comment:
❓ Why is it no longer private?

Reply:
Mistake. I'm adding it back in to the next commit.